### DISCUSSION TITLE
Enhancement of invload command

### DISCUSSION THREAD

**Discussion by MangoTang7** [Zrips/CMI#6992] [Enhancement]
Add command **/invsave [slots]** save the items in the specified backpack slot

For example, if I want to save the items of 0,3,5 slots of quickbar in my backpack,
I can use **/invsave 0,3,5** to save the items of the corresponding slots.
I hope it can be adopted. Please

**Comment by CloudeLecaw** [comment]
Like seen here
https://github.com/Zrips/CMI/issues/6991

Command syntax of suggeestion was corrected to
**cmi invsave (playerName) (id) (slot,slots) (-s)**

**Comment by MangoTang7** [comment]
ok! This is complete and great!
